eng/jacoco-test-coverage/pom.xml

```diff
@@ -386,7 +386,7 @@
     <dependency>
       <groupId>com.azure.spring</groupId>
       <artifactId>azure-spring-boot</artifactId>
-      <version>3.0.0</version> <!-- {x-version-update;com.azure.spring:azure-spring-boot;current} -->
+      <version>3.1.0-beta.1</version> <!-- {x-version-update;com.azure.spring:azure-spring-boot;current} -->
```

Comment:
It should be `3.2.0-beta.1`, because `3.0.x` and `3.1.x` will be reserved for spring-boot-2.2.x and spring-boot-2.3.x. Same to spring-cloud related modules.

Refs:
![image](https://user-images.githubusercontent.com/13167207/103491381-6e56b280-4e5e-11eb-8c7b-921a9d2ba322.png)

Reply:
This line is not updated. Is it a mistake?